.github/workflows/commit.yaml

```diff
@@ -97,6 +97,17 @@ jobs:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
         run: bash <(curl -s https://codecov.io/bash)
 
+      - name: "Change timezone" # https://github.com/tetratelabs/func-e/issues/303.
+        uses: szenius/set-timezone@v1.0
```

Comment:
Is this something we can rely on? I mean I want to avoid introducing fragile third party actions dep vs manually setting timezone😄

Reply:
We can do Linux only on this if you prefer that one. This deps basically do exec-command:

https://github.com/szenius/set-timezone/blob/master/index.js#L13-L24

Reply:
Thanks, only doing in Linux sounds reasonable!